docs/3.tutorials/nfts/9-series.md

```diff
@@ -0,0 +1,428 @@
+
+```diff
+pub owner_id: AccountId,
++ pub approved_minters: LookupSet<AccountId>,
++ pub approved_creators: LookupSet<AccountId>,
+pub tokens_per_owner: LookupMap<AccountId, UnorderedSet<TokenId>>,
+pub tokens_by_id: UnorderedMap<TokenId, Token>,
+- pub token_metadata_by_id: UnorderedMap<TokenId, TokenMetadata>,
++ pub series_by_id: UnorderedMap<SeriesId, Series>,
+pub metadata: LazyOption<NFTContractMetadata>,
+```
+
+Most of the information is the same, although we've added 2 new lookup sets and we've changed the `token_metadata_by_id` to be `series_by_id`.
+
+- **approved_minters**: Keeps track of accounts that can call the `nft_mint` function.
+- **approved_creators**: Keeps track of accounts that can create new series.
+- **series_by_id**: Map a series ID (u64) to its Series object.
+
+In addition, we're now keeping track of a new object called a `Series`.
+
+```rust
+pub struct Series {
+    // Metadata including title, num copies etc.. that all tokens will derive from
+    metadata: TokenMetadata,
+    // Royalty used for all tokens in the collection
+    royalty: Option<HashMap<AccountId, u32>>,
+    // Set of tokens in the collection
+    tokens: UnorderedSet<TokenId>,
+    // What is the price of each token in this series? If this is specified, when minting,
+    // Users will need to attach enough $NEAR to cover the price.
+    price: Option<Balance>,
+    // Owner of the collection
+    owner_id: AccountId,
+}
+```
+
+This object stores information that each token will inherit from. This includes:
+- The metadata.
+- The royalties.
+- The price.
+
+If a price is specified, there will be no restriction on who can mint tokens in the series. In addition, if the `copies` field is specified in the metadata, **only** that number of NFTs can be minted. If the field is omitted, an unlimited amount of tokens can be minted.
+
+We've also added a field `tokens` which keeps track of all the token IDs that have been minted for this series. This allows us to deal with the potential `copies` cap by checking the length of the set. It also allows us to paginate through all the tokens in the series.
+
+### Series File
+
+Moving on to a new file that has been introduced, you'll notice that there's no longer the `mint.rs` file and it's been replaced by a newer `series.rs` file. This is responsible for everything related to creating series as well as minting NFTs. Starting with the logic for creating a new series, you'll see a new function `create_series`.
+
+```rust reference
+https://github.com/near-examples/nft-tutorial/blob/main/nft-series/src/series.rs#L7-L58
+```
+
+The function takes in a series ID in the form of a `u64`, the metadata, royalties, and the price for tokens in the series. It will then create the `Series` object and insert it into the contract's `series_by_id` data structure. It's important to note that the caller must be an approved creator and they must attach enough $NEAR to cover storage costs.
```

Comment:
I would link `u64`, `Series` object, and `series_by_id` struct so people can have easy ways to learn more about something they don't yet understand. 

As much as we want users to follow a narrative that we create.... that can often not be the case as people skip to a section they want to learn about or (more likely a case) that people land on a section from a search.